ui/MultiSigDialogs.py

```diff
@@ -977,11 +975,11 @@ def createLockboxDashboardTab(self):
       # simulfunding widgets
       self.stkDashboard = QStackedWidget()
 
-      simultxt = 'Simul'
+      simultxt = self.tr('SimulFund')
```

Comment:
It should be left as SimulFund then, without a tr encapsulation. Sadly I believe translating a trademark without the owner's approval is a form of trademark law violation.